services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ConnectedDevicesSpec.groovy

```diff
@@ -877,4 +1069,16 @@ srcLldpDevices=#newSrcEnabled, dstLldpDevices=#newDstEnabled"() {
         //other non-mandatory lldp fields are out of scope for now. Most likely they are not properly parsed
         return true
     }
+
+    private static String getDescr(boolean src, boolean dst) {
+        if (src && !dst) {
+            "src only"
+        } else if (!src && dst) {
+            "dst only"
+        } else if (src && dst) {
+            "src and dst"
+        } else {
+            "none of the endpoints"
+        }
+    }
```

Comment:
We have to ignore all tests where the `multiTable` feature is required until #2971 is fixed.

Reply:
we are deleting via v1 instead. Is this an option?